### DISCUSSION TITLE
disable certain commands in a town

### DISCUSSION THREAD

**Discussion by stijnjw** [TownyAdvanced/Towny#4731] [Q&A]
Is it possible to disable certai commands inside a different town?
I'm trying to make it so people can't set their homes in different towns.

**Comment by LlmDl** [comment]
Towny only has the capability to disable commands run by jailed players currently.

For homes I recommend you set up an alias for the /home command in the commands.yml, to point /home to /res spawn.

Then set deny_bed_use true in your towny config.yml.

This will mean that players set their /home by using beds in plots they personally own, and in Inn plots.
